docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_independent_evaluation.yml

```diff
@@ -24,6 +24,9 @@ features:
   bootstrap theme: mac-bootstrap.css
   progress bar: True
 ---
+objects:
```

Comment:
Did you end up having to add the logo to every file and not just basic? I thought this was working already

Reply:
It should be just be in the basic. I will fix that